README.md

```diff
@@ -9,16 +9,16 @@ Instead of manually calling `use()` for every request, `use()` is called automat
 ## Example
 
 ```js
+var superagent = require('superagent'),
+    superagentUse = require('superagent-use'),
```

Comment:
I think we should not suggest to assign the original superagent to a variable; I prefer something like this:

```js
var superagent = require('superagent-use')(require('superagent')),
```

This would also avoid the rename to `request`  (see below)